res/skins/LateNight/aux.xml

```diff
@@ -0,0 +1,101 @@
+<Template>
```

Comment:
aux.xml is not a valid filename under windows.
https://msdn.microsoft.com/en-us/library/windows/desktop/aa365247%28v=vs.85%29.aspx#NAMING_CONVENTIONS

with that name, aux.xml will never be installed on windows, and even worst, I can not git pull anymore without errors :(
can you change the name of that file please ?
TIA
sb